docs/helm-values-documenation.md

```diff
@@ -12,6 +12,7 @@
 | `appgw.name` | | Name of the Application Gateway. Example: `applicationgatewayd0f0` |
 | `appgw.environment`| `AZUREPUBLICCLOUD` | Specify which cloud environment. Possbile values: `AZURECHINACLOUD`, `AZUREGERMANCLOUD`, `AZUREPUBLICCLOUD`, `AZUREUSGOVERNMENTCLOUD` |
 | `appgw.shared` | false | This boolean flag should be defaulted to `false`. Set to `true` should you need a [Shared App Gateway](setup/install-existing.md#multi-cluster--shared-app-gateway). |
+| `appgw.prefix` | No prefix if empty | Prefix that should be used in the naming of the Application Gateway's sub-resources|
```

Comment:
Hey sorry for late comment, do you think `subResourceNamePrefix` would be a more clear name ?

Reply:
Maybe will be better name, let me a moment to change it :)

Reply:
Hi @akshaysngupta 
Done! :)